Name not Defined

### Problem

When scheduling the Lambda function to run, this error is seen:
```
[ERROR] NameError: name 'name' is not defined
Traceback (most recent call last):
  File "/var/task/lambda_function.py", line 6, in lambda_handler
    wantedName = event.get(name)
```

JSON Input
```
{"name":"test", "token":"test", "delay":1}
```

--- a/ChangeMinecraftName.py
+++ b/ChangeMinecraftName.py
@@ -2,9 +2,9 @@ import urllib3, time
 
 def lambda_handler(event, context):
     https = urllib3.PoolManager()
-    wantedName = event.get(name)
-    token = event.get(token)
-    delay = event.get(delay)
+    wantedName = event.get('name')
+    token = event.get('token')
+    delay = event.get('delay')
     
     #Delay for specified seconds because Cloudwatch rule smallest increment of time is minutes
     time.sleep(delay)

